src/options/init.rs

```diff
@@ -8,6 +8,10 @@ pub struct Init {
     /// Name of the first fuzz target to create
     pub target: String,
 
+    #[arg(long, value_parser = clap::builder::BoolishValueParser::new(), default_value = "true")]
+    /// Whether to create a separate workspace for fuzz targets crate
+    pub create_workspace: Option<bool>,
```

Comment:
@fitzgen should we default to false then?

Reply:
Ah I guess this was about when there was a workspace already but then the fuzz crate wasn't listed in its deps?

Reply:
I don't think we should default this to `false` until we add support for adding the fuzz crate to the existing workspace.

Reply:
@fitzgen back then it needed to because it needed a different profile and rust flags

now this works with vanilla cargo

Reply:
My concern is about the "believes it is in a workspace when it is not" stuff that happens when there is an existing workspace, but cargo fuzz doesn't add the new fuzz crate to the existing workspace, as described in https://github.com/rust-fuzz/cargo-fuzz/issues/345#issuecomment-1613495598

Reply:
hmm yeah

Reply:
> `I don't think we should default this to false until we add support for adding the fuzz crate to the existing workspace.`

While not ideal, expecting users to do this edit manually is in line with what users currently have to do with `cargo new` command as can be seen in https://github.com/rust-lang/cargo/issues/8365

Reply:
I find that a compelling argument. In that light, I think we could default to not introducing a new workspace for the fuzz crate.

Reply:
I just added a separate commit with the default changed. Added a sample test case which simulates manual update of root `Cargo.toml`. Let me know what you think.

Reply:
I am not sure if `--create-workspace` is the best name for the parameter. Might be better to call it `--independent-workspace` or something else. It gets confusing since users might be thinking this relates to the root workspace.

Reply:
How about `--fuzzing-workspace`?

Reply:
Sounds good, `--fuzzing-workspace` is less ambiguous. Just renamed and force-pushed.